requirements.txt

```diff
@@ -1,9 +1,9 @@
 click==7.1.2
-google-api-core==1.20.1
+google-api-core==1.21.0
 googleapis-common-protos==1.52.0
 jinja2==2.11.2
 MarkupSafe==1.1.1
 protobuf==3.12.2
 pypandoc==1.5
 PyYAML==5.3.1
-dataclasses==0.7
\ No newline at end of file
+dataclasses==0.7; python_version < '3.7'
```

Comment:
This should be 0.6 instead, otherwise it will fail bazel build.

Reply:
🤦‍♀️ Fixed!